core/src/main/scala/org/apache/spark/sql/delta/metering/DeltaLogging.scala

```diff
@@ -91,19 +97,38 @@ trait DeltaLogging
   protected def recordDeltaOperation[A](
       deltaLog: DeltaLog,
       opType: String,
-      tags: Map[TagDefinition, String] = Map.empty)(
-      thunk: => A): A = {
+      tags: Map[TagDefinition, String] = Map.empty)(thunk: => A): A = {
     val tableTags = if (deltaLog != null) {
       Map(
         TAG_TAHOE_PATH -> Try(deltaLog.dataPath.toString).getOrElse(null),
         TAG_TAHOE_ID -> Try(deltaLog.snapshot.metadata.id).getOrElse(null))
     } else {
       Map.empty
     }
-    recordOperation(
-      new OpType(opType, ""),
-      extraTags = tableTags ++ tags) {
-          thunk
-    }
+    LoggerImplementation.activeLogger
```

Comment:
nit: can we retain the indentation as before

Reply:
Reverted the indentation.